NanoGardener/python/framework/samples/samplesCrossSections_UL.py

```diff
@@ -0,0 +1,395 @@
+# Cross section DB for Ultra-Legacy samples
+# Units in pb
+#
+# Detailed references at: https://docs.google.com/spreadsheets/d/1IEfle0H1V3ih2JVFpYckmTd-ACTBqgBRIsFydegGgPQ/edit?usp=sharing
+#
+# References
+#
+#	A	https://twiki.cern.ch/twiki/bin/view/CMS/StandardModelCrossSectionsat13TeV		
+#	B	https://twiki.cern.ch/twiki/bin/view/LHCPhysics/TtbarNNLO		
+#	C	https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt1314TeV		
+#	D	https://twiki.cern.ch/twiki/bin/view/LHCPhysics/SingleTopRefXsec		
+#	E	https://twiki.cern.ch/twiki/bin/view/CMS/SummaryTable1G25ns
+#       F       https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageAt13TeV
+#	F2	https://github.com/latinos/LatinoAnalysis/blob/master/Tools/python/HiggsXSection.py
+#	G	https://twiki.cern.ch/twiki/bin/view/CMS/GenXsecTaskForce		
+#	H	http://arxiv.org/pdf/1307.7403v1.pdf		
+#	I	https://twiki.cern.ch/twiki/bin/viewauth/CMS/HowToGenXSecAnalyzer		
+#	J	https://svnweb.cern.ch/cern/wsvn/LHCDMF/trunk/doc/tex/TTBar_Xsecs_Appendix.tex
+#	K	https://twiki.cern.ch/twiki/bin/view/CMS/HiggsWW13TeVProductionMassScan (powheg numbers)
+#	L	https://twiki.cern.ch/twiki/bin/view/CMS/HiggsWW13TeVProduction (powheg numbers)
+#       M       https://github.com/shu-xiao/MadGraphScanning/blob/master/diffCrossSection/madGraph.txt
+#       N       MCM
+#       O       https://twiki.cern.ch/twiki/pub/LHCPhysics/LHCHXSWG/Higgs_XSBR_YR4_update.xlsx
+#       P       https://drive.google.com/file/d/0B7mfFpGbPaMvb0ZtMlJfdXhJb2M/view
+#       Q       #https://indico.cern.ch/event/448517/session/0/contribution/16/attachments/1164999/1679225/Long_Generators_WZxsec_05_10_15.pdf
+#	R	https://cms-pdmv.cern.ch/mcm/requests?page=0&prepid=B2G-RunIISummer15GS*&dataset_name=TTbarDMJets_*scalar_Mchi-*_Mphi-10000_TuneCUETP8M1_13TeV-madgraphMLM-pythia8
+#       S       https://docs.google.com/spreadsheets/d/1b4qnWfZrimEGYc1z4dHl21-A9qyJgpqNUbhOlvCzjbE/edit?usp=sharing
+#       T       https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageBR
+#       U       https://twiki.cern.ch/twiki/pub/CMS/MonoHCombination/crossSection_ZpBaryonic_gq0p25.txt
+#	V	https://twiki.cern.ch/twiki/bin/viewauth/CMS/SameSignDilepton2016
+#       W       https://cms-gen-dev.cern.ch/xsdb/
+#       Z       http://cms.cern.ch/iCMS/analysisadmin/cadilines?line=SMP-18-006
+#       Y       https://twiki.cern.ch/twiki/bin/view/LHCPhysics/CERNYellowReportPageBSMAt13TeV
+#       A1      https://indico.cern.ch/event/673253/contributions/2756806/attachments/1541203/2416962/20171016_VJetsXsecsUpdate_PH-GEN.pdf 
+#	X	Unknown! - Cross section not yet there
+
+### W+jets
+samples['WJetsToLNu']                   	.extend( ['xsec=61526.7',	'kfact=1.00',		'ref=E'] )
```

Comment:
It should be `Samples` and not `samples` if I'm not wrong.

Reply:
In the master branch, the old samplesCrossSections files used `samples`:
https://github.com/latinos/LatinoAnalysis/blob/master/NanoGardener/python/framework/samples/samplesCrossSections2016.py

It is true that in the samples list files, we use `Samples`: 
https://github.com/latinos/LatinoAnalysis/blob/master/NanoGardener/python/framework/samples/Summer16_102X_nAODv7.py

I just kept the same convention.

Reply:
Right, sorry for the misunderstanding... I think the issue is that there is a mixed usage of `Samples` and `samples` in this file.